src/Navigation.jsx

```diff
@@ -0,0 +1,26 @@
+import { Link } from 'react-router-dom';
+
+const Navigation = () => {
+
+    const NavigationConfig = [
+        {label: 'Country', link: '/'},
+        {label: 'States', link: '/states'},
+        {label: 'Cities', link: '/cities'},
+        {label: 'Cards', link: '/cards'},
+        {label: 'Login', link: '/login'},
+    ]
```

Comment:
Please, create a common paths map, that you'll be able to use here and in your routes
Additionally, please, move it out of the Navigation component as is the constant